bootloader/assets/grub_test.go

```diff
@@ -0,0 +1,62 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package assets_test
+
+import (
+	"bytes"
+	"testing"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/bootloader/assets"
+	"github.com/snapcore/snapd/testutil"
+)
+
+// Hook up check.v1 into the "go test" runner
+func Test(t *testing.T) { TestingT(t) }
+
+type grubAssetsTestSuite struct{}
+
+var _ = Suite(&grubAssetsTestSuite{})
+
+func (s *grubAssetsTestSuite) testGrubConfig(c *C, name string, keys []string) {
```

Comment:
The name does't really indicate what is tested, perhaps this simple change would do.
You could also use `...` instead of `[]string` to simplify the test code call site.
```suggestion
func (s *grubAssetsTestSuite) testGrubConfigContains(c *C, name string, keys []string) {
```